src/shared/json.h

```diff
@@ -332,8 +332,8 @@ typedef enum JsonDispatchFlags {
         JSON_RELAX      = 1 << 4, /* Use relaxed user name checking in json_dispatch_user_group_name */
 
         /* The following two may be passed into log_json() in addition to the three above */
-        JSON_DEBUG      = 1 << 4, /* Indicates that this log message is a debug message */
-        JSON_WARNING    = 1 << 5, /* Indicates that this log message is a warning message */
+        JSON_DEBUG      = 1 << 5, /* Indicates that this log message is a debug message */  //
```

Comment:
what's that `//` at the end?

Reply:
It was my mark to mark that this needs fixing, which of course I forgot to remove ;)